raster_loader/io/common.py

```diff
@@ -495,8 +507,14 @@ def raster_band_approx_stats(
     _sum = 0
     sum_squares = 0
     if count > 0:
-        _sum = int(np.sum(samples_band))
-        sum_squares = int(np.sum(np.array(samples_band) ** 2))
+        try:
+            _sum = int(np.sum(samples_band))
+        except OverflowError:
+            _sum = float("inf")
```

Comment:
2 questions:
- why is `inf` the result of the sum in that geotiff?
- Why do we want to set the value as `inf` when the overflow error rises? I know that the result is `inf` but, is it useful? isn't it better `None` or zero? if you set the sum as inf in the next append iteration the result will be inf again...

Reply:
The `inf` results is due to the fact that the testing geotiff has changed the `nodata` parameter to `0` but not having changed the actual `nodata` cell values that remain set to `-3.4028230607370965e+38`.

I have already changed this and now `_sum` and `sum_squares` are set to `0` when the sum  is overflown.